loadtest/control/simplecontroller/actions.go

```diff
@@ -88,3 +88,20 @@ func (c *SimpleController) createGroupChannel() user.UserStatus {
 	
 	return user.UserStatus{User: c.user, Info: fmt.Sprintf("group channel created, id %v", channelId)}
 }
+
+func (c *SimpleController) viewChannel() user.UserStatus {
+	channel, err := c.user.Store().Channel("") // TODO: fetch channel randomly?
```

Comment:
Correct, here we are going to use [this](https://mattermost.atlassian.net/browse/MM-20704).